OloEngine/src/OloEngine/Audio/SoundGraph/SoundGraphSound.cpp

```
@@ -395,24 +424,32 @@ namespace OloEngine::Audio::SoundGraph
         // event: Acquire drives OnVoiceStart synchronously when this voice wins a slot, and
         // that is where the gain is un-muted. Starting a re-triggered sound also drops any
         // previous registration so the budget never holds two records for one graph.
-        ReleaseVoice(/*restoreGain=*/true);
+        ReleaseVoice(/*resumePlayback=*/true);
         const auto handle = OloEngine::Audio::VoiceManager::Get().Acquire(this, BuildVoiceParams());
         m_VoiceHandle.store(handle, std::memory_order_release);
         if (handle == OloEngine::Audio::kInvalidVoiceHandle)
         {
```

Cppcheck (2.21.0)

[style] The function 'AddNode' is never used.

(unusedFunction)